src/test/expression/expression_evaluator_to_values_test.cpp

```diff
@@ -289,6 +313,17 @@ TEST_F(ExpressionEvaluatorToValuesTest, PredicatesSeries) {
   EXPECT_TRUE(test_expression<int32_t>(table_a, *between_(b, a, c), {1, std::nullopt, 1, std::nullopt}));
   EXPECT_TRUE(test_expression<int32_t>(table_a, *between_(e, a, f), {1, 0, 0, 0}));
   EXPECT_TRUE(test_expression<int32_t>(table_a, *between_(3.3, a, b), {0, 0, 1, 0}));
+  EXPECT_TRUE(
```

Comment:
(III) the tests you added would pass even if all between types were rewritten to the same expression. Please write tests that actually test the lower and upper bound.

Reply:
what about now?